spec/bolt/executor_spec.rb

```diff
@@ -177,7 +178,7 @@ def mock_node_results(_run_as = nil)
       node_results.each do |target, result|
         expect(ssh)
           .to receive(:run_task)
-          .with(target, task_type(task), task_arguments, {})
+          .with(target, task_type(task), task_arguments, _task)
```

Comment:
I think just rewrite the _task options here

Reply:
What's wrong with having it as a variable?

Reply:
You're not using it consistently and it's weirdly named. I would expect `_task` to be `service::restart` not an opts hash containing only the `_task` options